cost/kubecost/cluster/kubecost_cluster_rightsizing_recommendations.pt

```diff
@@ -36,6 +36,22 @@ parameter "param_kubecost_host" do
   # No default value, user input required
 end
 
+parameter "param_allow_shared_core" do
+  type "string"
+  category "Allow Shared Core"
+  label "Shared core"
+  allowed_values "true", "false"
+  description "Enable or disable the use of shared cores in calculating costs and resource savings in Kubecost"
+end
+
+parameter "param_include_overhead" do
+  type "string"
+  category "Include overhead"
+  label "Include overhead"
+  allowed_values "true", "false"
+  description "By enabling this option, Kubecost takes into account not only the resources used by applications and containers, but also those needed to operate and maintain the Kubernetes environment"
```

Comment:
We might want to improve the description here.

Reply:
We have changed by one *** Whether to include resources for operating and maintaining the Kubernetes environment, in addition to those used by applications and containers.***

We don't have much information about overhead but we saw before on Kubecost UI send this flag as false in order to have coherence with UI and  our incidents  we decide to add this flag. Kubecost said this about this concept: [link](https://docs.kubecost.com/using-kubecost/navigating-the-kubecost-ui/assets)

> Overhead costs refer to the capacity costs minus allocatable costs. This includes Kubelet, container runtime, and node operating systems and software. It does not include Prometheus, DNS, cert-manager, kube-system, or pods running with Kubernetes. To learn more about overhead costs, watch [Node Size Matters](https://www.youtube.com/watch?v=6vNI_O6sdvY), a presentation by Kubecost engineers from KubeCon 2023.

And on the video mentioned, talks about Overhead as: 

....  overhead in the Kubernetes environment often refers to the resources consumed beyond what is directly allocated to user applications, such as maintaining the cluster and running system components.

Reply:
done we modified  the description